bigquery/indexer.py

```diff
@@ -241,12 +241,20 @@ def index_table(es, bq_client, storage_client, index_name, table,
         scripts_by_id = _sample_scripts_by_id_from_export(
```

Comment:
We no longer need scripts so in general we can get rid of indexer_util.bulk_index_scripts, indexer.py UPDATE_SAMPLES_SCRIPT, etc. You would still need special logic for samples, the special logic just shouldn't involve "scripts" (which is an Elasticsearch thing).